picasso/src/main/java/com/squareup/picasso3/Picasso.java

```diff
@@ -660,13 +669,28 @@ void cancelExistingRequest(Object target) {
 
     private boolean indicatorsEnabled;
     private boolean loggingEnabled;
+    private boolean isChild;
 
     /** Start building a new {@link Picasso} instance. */
     public Builder(@NonNull Context context) {
       checkNotNull(context, "context == null");
       this.context = context.getApplicationContext();
     }
 
+    Builder(Picasso picasso) {
+      context = picasso.context;
+      callFactory = picasso.callFactory;
+      service = picasso.dispatcher.service;
+      cache = picasso.cache;
+      listener = picasso.listener;
+      requestTransformers.addAll(picasso.requestTransformers);
+      requestHandlers.addAll(picasso.extraRequestHandlers);
```

Comment:
Nit: could we do a subList call here on the requestHandlers instead of making the extra field? I think that's what Moshi and Retrofit do.

Reply:
Ok.  This feels like one of those things that will be easy to forget to update..

Reply:
Test case then? We have ones in Retrofit at least to ensure the default ones don't get copied twice.

Reply:
Yup, there's a test case here: https://github.com/square/picasso/blob/aa6005e3115c48b1e411cb1c5a14272b111601b2/picasso/src/test/java/com/squareup/picasso3/PicassoTest.java#L565.

Thoughts?

Reply:
ping @JakeWharton